arrow-meta/build.gradle

```diff
@@ -31,7 +31,7 @@ dependencies {
     testCompile project(":arrow-meta:arrow-meta-test-models")
     testCompile("junit:junit:$JUNIT_VERSION")
     testRuntime("org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION")
-    testCompile "io.kotlintest:kotlintest-runner-junit4:$KOTLIN_TEST_VERSION", excludeArrow
+    testCompile "io.kotest:kotest-runner-junit5-jvm:$KOTLIN_TEST_VERSION", excludeArrow
```

Comment:
I think it's just necessary to remove `excludeArrow` here because `testCompile` dependency.